docarray/array/storage/sqlite/getsetdel.py

```diff
@@ -26,8 +26,8 @@ def _set_doc_by_offset(self, offset: int, value: 'Document'):
 
     def _set_doc_by_id(self, _id: str, value: 'Document'):
         self._sql(
-            f'UPDATE {self._table_name} SET serialized_value=? WHERE doc_id=?',
-            (value, _id),
```

Comment:
What's the consequence after: 

```
da['a'] = Document(id='c', text='c')
```

In your commit, `da['a']` is still available, and not updated, am I right?